sanghyeok/LV_1/3진법-뒤집기.js

```diff
@@ -0,0 +1,18 @@
+function solution(n) {
+    let result = 0;
+    let conver = '';
+    
+    while(n > 0) {
+        conver += n % 3;
+        n = parseInt(n / 3);  
```

Comment:
```suggestion
        n.toString(3)
```
toString 메서드를 활용하면 3진법으로 쉽게 변환할 수 있습니다! 이 메서드와 다른 메서드 하나를 조합하면 한줄로도 처리할 수 있으니 사용해 보시면 좋을 것 같습니다👍

Reply:
네 toString() 메서드 활용으로 좀 더 깔끔한 코드를 만드는 점에 숙지해야겠습니다. 피드백 감사합니다!!